benchmarks/benchmarks/optimize.py

```diff
@@ -506,7 +507,7 @@ class BenchGlobal(Benchmark):
         _enabled_functions = list(_functions.keys())
 
     params = [
-        list(_functions.keys()),
+        _enabled_functions,
```

Comment:
There's no need to add all the functions to the list of params, only send those you want to run.